pytest-selenium/tests/test_toc.py

```diff
@@ -18,17 +19,23 @@ def test_toc_toggle_button_opens_and_closes(selenium, base_url, book_slug, page_
 
     """
     # GIVEN: The selenium driver, base_url, book_slug, and page_slug
+    # WHEN: The book and page URL is loaded
     content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
 
     toolbar = content.toolbar
     sidebar = content.sidebar
+    toc = content.sidebar.toc
 
     # AND: Window width is 1024 or greater (Desktop)
     if content.is_desktop:
 
-        # Sidebar is open by default
+        # THEN: Sidebar is open by default
         assert sidebar.header.is_displayed
 
+        # AND: The page name in the sidebar is bolded to indicate that its selected
+        bold = "400"
+        assert toc.font_property_of_selected_section == bold
```

Comment:
probably want this to be more like `is_bold(toc.active_section)` and make an `is_bold` helper somewhere, maybe like a place to be a collection of font related assertions